### PR TITLE
Add more space level metrics

### PR DESCRIPTION
**Pull request by jievince**
Co-authored-by: Yee <2520865+yixinglu@users.noreply.github.com>
Co-authored-by: Sophie <84560950+Sophie-Xie@users.noreply.github.com>

#### What type of PR is this?
- [ ] bug
- [x] feature
- [ ] enhancement

#### Checklist:
- [ ] Documentation affected (Please add the label if documentation needs to be modified.)
- [ ] Incompatibility (If it breaks the compatibility, please describe it and add the corresponding label.）
- [ ] If it's needed to cherry-pick (If cherry-pick to some branches is required, please label the destination version(s).)
- [ ] Performance impacted: Consumes more CPU/Memory

#### Release notes:

Please confirm whether to be reflected in release notes and how to describe:
>                                                                 `

### CODE REVIEW

**Review by czpmango — APPROVED** [review]

File: src/graph/stats/GraphStats.h [hunk 1]
```diff
@@ -23,14 +23,14 @@ extern stats::CounterId kNumSentences;
 extern stats::CounterId kQueryLatencyUs;
 extern stats::CounterId kSlowQueryLatencyUs;
 extern stats::CounterId kNumKilledQueries;
+extern stats::CounterId kNumOomQueries;
```
Comment (czpmango) [inline src/graph/stats/GraphStats.h:26]: Rename `kNumOomQueries`, maybe `kNumQueriesHitMemoryWatermark`?

File: src/graph/executor/Executor.cpp [hunk 2]
```diff
@@ -155,16 +155,25 @@ Executor *Executor::makeExecutor(const PlanNode *node,
...
   switch (node->kind()) {
     case PlanNode::Kind::kPassThrough: {
       return pool->add(new PassThroughExecutor(node, qctx));
     }
     case PlanNode::Kind::kAggregate: {
       stats::StatsManager::addValue(kNumAggregateExecutors);
+      if (FLAGS_enable_space_level_metrics && spaceName != "") {
+        stats::StatsManager::addValue(
+            stats::StatsManager::counterWithLabels(kNumAggregateExecutors, {{"space", spaceName}}));
+      }
```
Comment (Aiee) [inline src/graph/executor/Executor.cpp:168]: Seems this part could be extracted as a method to omit redundant code.

Comment (jievince) [reply src/graph/executor/Executor.cpp:168]: It couldn't indeed.

File: src/graph/stats/GraphStats.h (hunk 1 above)
Comment (jievince) [reply src/graph/stats/GraphStats.h]: ok

**Review by Aiee — APPROVED** [review]

**Review by nevermore3 — APPROVED** [review]

**Pull request merged**